File: 프로그래머스/0/120815. 피자 나눠 먹기 （2）/피자 나눠 먹기 （2）.java
class Solution {
    public int solution(int n) {
        int answer = 1; // 최소 피자 판 수 초기화

        // 최소한의 피자 판 수를 찾을 때까지 반복
        while ((answer * 6) % n != 0) {
            answer++; // 조건을 만족할 때까지 answer 증가
        }

        return answer; // 계산된 최소 피자 판 수 반환
    }

    public static void main(String[] args) {
        Solution sol = new Solution();

        // 테스트 예제 실행
        System.out.println(sol.solution(6));  // 1
        System.out.println(sol.solution(10)); // 5
        System.out.println(sol.solution(4));  // 2
    }
}

File: 프로그래머스/0/120815. 피자 나눠 먹기 （2）/피자 나눠 먹기 （2）.js
function solution(n) {
    var answer = 0;
    
    for (let i = 1; i < n * 6; i++) {
        if (6 * i % n == 0) {
            answer = i;
            break;
        }
    }
    
    return answer;
}

console.log(solution(6));
console.log(solution(10));
console.log(solution(4));